lib/models/lookup.js

```diff
@@ -74,37 +75,46 @@ function LookupModelFactory (Model, assert, Errors, Promise, Constants) {
         setIp: function(ipAddress, macAddress) {
             var self = this;
 
-            return Promise.all([
-                self.findOne({ ipAddress: ipAddress }),
-                self.findOne({ macAddress: macAddress })
-            ]).spread(function(ipRecord, macRecord) {
-                if (macRecord && macRecord.ipAddress === ipAddress) {
-                    throw new Errors.BreakPromiseChainError();
-                }
-                return [ipRecord, macRecord];
-            }).spread(function(ipRecord, macRecord) {
-                if (!ipRecord) {
-                    return [macRecord];
-                }
-                if (ipRecord.ipAddress === ipAddress && ipRecord.macAddress !== macAddress) {
-                    return [
-                        macRecord,
-                        self.destroy({ id: ipRecord.id })
-                            .then(function() {
-                                delete ipRecord.ipAddress;
-                                return self.create(ipRecord);
-                            })
-                    ];
-                }
-            }).spread(function(macRecord) {
-                if (!macRecord) {
-                    return self.create({ ipAddress: ipAddress, macAddress: macAddress });
-                } else {
-                    return self.update({ id: macRecord.id }, { ipAddress: ipAddress });
+            var query = {
+                ipAddress: ipAddress, // 10.1.1.2
```

Comment:
the specific ip address'10.1.1.2' is confused. could you help refine the annotation?